### PR TITLE
Replace user operation

### PR DESCRIPTION
**Pull request by Vid201**
Closese #100

### CODE REVIEW

File: src/bundler/mod.rs [hunk 1]
```diff
@@ -138,6 +138,9 @@ impl Bundler {
         let tx_hash = tx.tx_hash();
         trace!("Send bundle with transaction: {tx:?}");
 
+        // wait some time, otherwise sometimes events for this bundle are not yet detected
+        thread::sleep(Duration::from_millis(25));
```
Comment (zsluedem) [inline src/bundler/mod.rs:142]: use `tx.await` ?

File: src/uopool/sanity_check.rs [hunk 2]
```diff
@@ -13,6 +13,14 @@ use ethers::{
 
 use super::UoPool;
 
+const MAX_UOS_PER_UNSTAKED_SENDER: usize = 4;
+const GAS_INCREASE_PERC: u64 = 10;
```
[the remark this answers is not in the document]
Comment (Vid201) [reply src/uopool/sanity_check.rs:17]: Yeah, they are not as far as I know (only in bundler spec tests).

This parameters could probably be configurable by each bundler in the future though, I don't think they have to be explicitly defined in the ERC-4337.

I think we can have it set this way for now, and see later what to do with them (or just leave them like that).

File: src/bundler/mod.rs (hunk 1 above)
Comment (Vid201) [reply src/bundler/mod.rs]: Yeah, that's better. I updates the polling interval for the pending transaction from 7 secs to 0.1, so 1 test (that contains sending bundle) is not running for 7 seconds +.

**Review by zsluedem — APPROVED** [review]
LGTM

**Pull request merged**